bundles/org.openhab.binding.miio/src/main/resources/OH-INF/thing/vacuumThing.xml

```diff
@@ -215,7 +215,41 @@
 		<label>Robot Locating</label>
 		<state readOnly="true"/>
 	</channel-type>
-
+	<channel-type id="dock_state">
+		<item-type>String</item-type>
+		<label>Dock State</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="dock_state_id">
+		<item-type>Number</item-type>
+		<label>Dock State ID</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="mop_drying_time">
+		<item-type>Number:Time</item-type>
+		<label>Mop Drying Time</label>
+		<state pattern="%.0f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="is_mop_drying">
+		<item-type>Switch</item-type>
+		<label>Mop Cleaning Active</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="collect_dust">
+		<item-type>Switch</item-type>
+		<label>Start collecting dust</label>
```

Comment:
```suggestion
		<label>Start Collecting Dust</label>
```

Reply:
I'm really struggling, when you want words in first letter upper case..  :-(